locale/en/blog/release/v19.6.1.md

```diff
@@ -0,0 +1,109 @@
+---
+date: 2023-02-16T22:21:38.949Z
+version: 19.6.1
+category: release
+title: Node v19.6.1 (Current)
+slug: node-v19-6-1
+layout: blog-post.hbs
+author: Rafael Gonzaga
+---
+
+### Notable Changes
+
+The following CVEs are fixed in this release:
+
+* **[CVE-2023-23919](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-23919)**: OpenSSL errors not cleared in error stack (Medium)
+* **[CVE-2023-23918](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-23918)**: Experimental Policies bypass via `process.mainModule.require`(High)
+* **[CVE-2023-23920](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-23920)**: Insecure loading of ICU data through ICU\_DATA environment variable (Low)
+
+More detailed information on each of the vulnerabilities can be found in [February 2023 Security Releases](https://nodejs.org/en/blog/vulnerability/february-2023-security-releases/) blog post.
+
+This security release includes OpenSSL security updates as outlined in the recent
+[OpenSSL security advisory](https://www.openssl.org/news/secadv/20230207.txt) and `undici` security update.
```

Comment:
Should there be a link to the undici blog post/release/etc. ?

Reply:
It will be included in the february-2023-secure-releases as far as I can tell.